tests/nosetests/test_legacy_parser/test_legacy_export_import.py

```diff
@@ -38,18 +32,15 @@ def test_reimport(filename=None):
         if pars[x].qualifier not in ['times', 'fluxes', 'sigmas', 'rvs']:
             
             try:
-                assert(val1==val2)
+                assert val1 == val2
             except:
-                assert(all(val1==val2))
-
-
-
+                assert all(val1==val2)
```

Comment:
not necessary here... but why is there a try/except?  Can we just check for array vs float explicitly?